src/main/java/edu/ucsb/mapache/documents/ChannelPurpose.java

```diff
@@ -0,0 +1,69 @@
+package edu.ucsb.mapache.documents;
+import java.util.Objects;
+
+public class ChannelPurpose {
+    private String value;
+    private String creator;
+
+    public ChannelPurpose() {
+    }
+
+    public ChannelPurpose(String value, String creator) {
+        this.value = value;
+        this.creator = creator;
+    }
+
+    public String getValue() {
+        return this.value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public String getCreator() {
+        return this.creator;
+    }
+
+    public void setCreator(String creator) {
+        this.creator = creator;
+    }
+
+   
+    @Override
+    public boolean equals(Object o) {
+        if (o == this)
+            return true;
+        if (!(o instanceof ChannelPurpose)) {
+            return false;
+        }
+        ChannelPurpose channelPurpose = (ChannelPurpose) o;
+        return Objects.equals(value, channelPurpose.value) && Objects.equals(creator, channelPurpose.creator);
```

Comment:
We should use `EqualsBuilder` over `Object.equals` because it results in less branches to test.

Reply:
Done